monitoring/cluster_observability_operator/cluster-observability-operator-overview.adoc

```diff
@@ -0,0 +1,29 @@
+:_content-type: ASSEMBLY
+[id="cluster-observability-operator-overview"]
+= Cluster Observability Operator overview
+include::_attributes/common-attributes.adoc[]
+:context: cluster_observability_operator_overview
+
+toc::[]
+
+:FeatureName: Using the Cluster Observability Operator
+include::snippets/technology-preview.adoc[leveloffset=+2]
+
+The Cluster Observability Operator (COO) is an optional component of the {product-title}. You can deploy it to create standalone monitoring stacks that are independently configurable for use by different services and users.
```

Comment:
```suggestion
The Cluster Observability Operator (COO) is an optional component of {product-title}. You can deploy it to create stand-alone monitoring stacks that are independently configurable for use by different services and users.
```
Per IBM SG re: "stand-alone"

Reply:
Thanks, Alex. Per our SSG, RH docs use "standalone": https://redhat-documentation.github.io/supplementary-style-guide/#standalone, so leaving as-is.